src/Node/Render.php

```diff
@@ -44,9 +68,12 @@ protected function addTemplateArguments(Twig_Compiler $compiler) {
   }
 
   /**
-   * Returns the component context from the component config file.
+   * Returns the component context variables from its config file.
    *
    * @todo Merge specific keys to prevent config duplication.
+   *
+   * @param array $component
+   *   The component context variables.
    */
   public function getComponentContext($component) {
```

Comment:
In the code of this function, you can use the `+=` operator to append non-existing keys to an array.

```diff
-$context = $context + $this->getVariantContext($component['variant'], $config);
+$context += $this->getVariantContext($component['variant'], $config);
```